hazelcast/src/main/java/com/hazelcast/ringbuffer/impl/operations/ReadManyOperation.java

```diff
@@ -53,20 +53,20 @@ public ReadManyOperation(String name, long startSequence, int minSize, int maxSi
 
     @Override
     public void beforeRun() {
-        RingbufferContainer ringbuffer = getRingBufferContainer();
-        ringbuffer.checkBlockableReadSequence(startSequence);
```

Comment:
Do we not check for too-large sequence because we're about to clamp to the largest anyway?

Reply:
Yes.